libethdrivers/CMakeLists.txt

```diff
@@ -117,4 +117,4 @@ string(
     "--undefined=odroidc2_ethernet_ptr"
 )
 
-target_link_libraries(ethdrivers "${DriverModules}")
+target_link_libraries(ethdrivers virtio "${DriverModules}")
```

Comment:
It's probably better to use:
`target_include_directories(ethdrivers PRIVATE $<TARGET_PROPERTY:virtio,INTERFACE_INCLUDE_DIRECTORIES>)`

This more explicitly only uses the library for the header files it exports.